.github/workflows/ghpages.yml

```diff
@@ -1,9 +1,9 @@
 name: Build and GH Pages Deploy
 
 on:
-  push:
-    branches:
-      - master
+  release:
+    types: [created]
```

Comment:
Should we also make this on tag push?

Reply:
we could, I just kind of wanted to test this event as well in some way. Perhaps try this for now and iterate? @rwwagner90